siliconcompiler/core.py

```diff
@@ -513,7 +523,11 @@ def create_cmdline(self,
 
         # Set loglevel if set at command line
         if 'option_loglevel' in cmdargs.keys():
-            self.logger.setLevel(cmdargs['option_loglevel'])
+            log_level = cmdargs['option_loglevel']
+            if isinstance(log_level, list):
+                # if multiple found, pick the first one
+                log_level = log_level[0]
+            self.logger.setLevel(log_level.split()[-1])
```

Comment:
Why do we have to split this?
I thought loglevel allowed values are:
`"NOTSET", "INFO", "DEBUG", "WARNING", "ERROR", "CRITICAL"`

Reply:
if set on a pernode basis the value might look like: `syn DEBUG` or `place 0 INFO`

Reply:
Then does the logging actually switch between different granularity levels per index?

Reply:
it should

Reply:
if not, then thats a bug